.test-infra/jenkins/job_PostCommit_Python_Sickbay.groovy

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import PostcommitJobBuilder
+
+import static PythonTestProperties.ALL_SUPPORTED_VERSIONS
```

Comment:
let's make sure that these tests run on a schedule (maybe every 12 hours or so)

Reply:
Yes. This test run each 6 hours , because use the postcommit job builder 
`commonJobProperties.setAutoJob delegate, '0 */6 * * *', 'builds@beam.apache.org', true, true`